modules/network-observability-auth-multi-tenancy.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-auth-mutli-tenancy_{context}"]
+= Configure authorization for multi-tenancy
```

Comment:
Could we rename as `Configure authorization and multi-tenancy` ? The reason is that users must still configure these authorizations even if they aren't interested in mutli-tenancy.
Multi-tenancy really starts when users run the below command "$ oc adm policy add-cluster-role-to-user netobserv-reader user1"; but in any case they still need to create the yaml